makefile/.konveyor/ansible/tests.mk

```diff
@@ -0,0 +1,15 @@
+### Important, the LANG_DIR must match one from Makefile
+LOCAL_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
+LOCAL_LANG_DIR = $(notdir $(patsubst %/,%,$(dir $(LOCAL_PATH))))
+
+$(LOCAL_LANG_DIR)-tests:
+	$(MAKE) pre-commit
+
+pre-commit: setup-pre-commit
+	. ${PYTHON_VENV}/bin/activate && pre-commit
+
+isort: setup-pre-commit
+	. ${PYTHON_VENV}/bin/activate && pre-commit run isort
```

Comment:
is this really target for ansible based projects?

Reply:
Same as above, removing. Was pasted along with the dir...